homeassistant/components/recorder/__init__.py

```diff
@@ -1028,3 +1028,20 @@ def _shutdown(self):
         self.hass.add_job(self._async_stop_queue_watcher_and_event_listener)
         self._end_session()
         self._close_connection()
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "recorder/info",
+    }
+)
+@callback
+def ws_info(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict,
+) -> None:
+    """Handle get info command."""
+    connection.send_result(
+        msg["id"], {"entity_filter": hass.data[DATA_INSTANCE].entity_filter.config}
```

Comment:
I'm not sure what the use case is going to be for this so this might not be a concern.

I'm wondering if it makes sense to ask the backend if a list/set of entities if filtered or not in case we end up changing how the filter is built on the backend and break the expectations on the frontend.

Reply:
That's a good point and an improvement we should make. I will do that when we're going to use it.